runtime/src/main/java/io/micronaut/jackson/serialize/OptionalValuesSerializer.java

```diff
@@ -35,13 +38,18 @@
 @Singleton
 public class OptionalValuesSerializer extends JsonSerializer<OptionalValues<?>> {
 
+    @Inject
+    private JacksonConfiguration jacksonConfiguration;
```

Comment:
private field injection requires reflection, please change this to a field representing the configuration and use constructor injection and then add an additional constructor to maintain compatibility. Something like:

```java
private final boolean alwaysSerializeErrorsAsList;

@Inject
public OptionalValuesSerializer(JacksonConfiguration jacksonConfiguration) {
       this. alwaysSerializeErrorsAsList = jacksonConfiguration.isAlwaysSerializeErrorsAsList();
}

public OptionalValuesSerializer() {
      this.alwaysSerializeErrorsAsList = true
}
```